libs/portals/admin/service-desk/src/screens/Users/Users.tsx

```diff
@@ -87,14 +87,12 @@ const Users = () => {
             buttonProps={{
               type: 'submit',
               disabled: searchInput.length === 0,
-              onClick: (e) => {
```

Comment:
This is not directly related to this pr, just a bug i noticed regarding changes i made last week and fixed